app/poros/item.rb

```diff
@@ -0,0 +1,11 @@
+class Item
+  attr_reader :id, :name, :description, :unit_price, :merchant_id
+
+  def initialize(data)
+    @id = data[:id].to_i
+    @name = data[:attributes][:name]
+    @description = data[:attributes][:description]
+    @unit_price = data[:attributes][:unit_price]
+    @merchant_id = data[:attributes][:merchant_id]
+  end
+end
```

Comment:
Careful with naming an attribute "id", since POROs automatically have a unique id associated with them. This can cause some issues later on when you're trying to access the id that you manually assigned in the initialize method.